modules/docs/src/main/mdoc/docs/mtl-combinators.md

```diff
@@ -99,10 +99,14 @@ retry on all errors.
 The API (modulo some type-inference trickery) looks like this:
 
 ```scala
-def retryingOnSomeErrors[M[_]: Monad, A, E: Handle[M, *]](
+def retryingOnAllErrors[M[_], E](
```

Comment:
The correct method signature here is

```
def retryingOnSomeErrors[M[_]: Monad: Sleep, A, E: Handle[M, *]](
  policy: RetryPolicy[M],
  onError: (E, RetryDetails) => M[Unit]
)(action: => M[A]): M[A]
```

The method name was wrong and the `Sleep` context bound was missing.

This combinator is part of the module that integrates with cats-mtl, so it uses the `Handle` type class instead of `MonadError`.